logs_sdk/lib/opentelemetry/sdk/logs/log_record_data.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module SDK
+    module Logs
+      # LogRecordData is a Struct containing {LogRecord} data for export.
+      LogRecordData = Struct.new(:timestamp,             # optional Integer nanoseconds since Epoch
+                                 :observed_timestamp,    # Integer nanoseconds since Epoch
```

Comment:
The `timestamp` and `observed_timestamp` are documented here to be "Integer nanoseconds since Epoch", so there should be no need for the `unix_nano_timestamp` and `unix_nano_observed_timestamp` methods below.

Reply:
Great point. Fixed in 0fb4606e20c8d71d4f2c47a71ac59898f4deb2e1